New folder/content/js/index.js

```diff
@@ -0,0 +1,48 @@
+const btnAddInput = document.getElementById('btnAdd');
+const btnDisplay = document.getElementById('btnDisplay');
+const elementsInput = document.querySelector('.elements');
+
+var elementList = [];
+
+if(localStorage.getItem("information") !== null){
```

Comment:
@Noha-sherif12 why you are using local storage

Reply:
to store the values the user entered